src/main/scala/mesosphere/marathon/api/ModelValidation.scala

```diff
@@ -288,3 +288,5 @@ trait ModelValidation extends BeanValidation {
   }
 
 }
+
+object ModelValidation extends ModelValidation
```

Comment:
No but in my opinion it should be used everywhere.

I do not like using traits like this. Do you?